### DISCUSSION TITLE
Extend the end of replays

### DISCUSSION THREAD

**Discussion by monochrome22** [ppy/osu#22481] [Ideas] [answered]
One thing that always bothered me in lazer is that replays end immediately after pressing the last object. In stable, it will continue recording for a few seconds, until the player goes to the results screen. Often during the few seconds after the map ends, players do movements with their cursors or even use smoke. 

Here are some examples: 
(rrtyui) [https://youtu.be/17PP3XB1IzM?t=131](https://youtu.be/17PP3XB1IzM?t=131) (skip to 2:11)
(Bikko) [https://youtu.be/xQmuYsXmBH4?t=135](https://youtu.be/xQmuYsXmBH4?t=135) (skip to 2:15)
(Cookiezi) [https://youtu.be/86pK4kJ1v_g?t=55](https://youtu.be/86pK4kJ1v_g?t=55) (skip to 0:55)
(aetrna) [https://youtu.be/hJmXbwF55rM?t=195](https://youtu.be/hJmXbwF55rM?t=195) (skip to 3:15)

While this is how it works in lazer:
[https://youtu.be/H2FeD34Pfew?t=85](https://youtu.be/H2FeD34Pfew?t=85) (skip to 1:25)

I feel that the movements that players do at the end of the replay add charm and personality to both the player and the play. I always thought it's cool how rrtyui flicks his cursor to the top of his screen, and seeing Bikko do that smooth curve after spinner adds a sense of calmness to his plays. Having the replay end immediately takes these away and makes it more dull.

I don't have suggestions on exactly how it should be implemented. Just have it record for a few extra seconds or until players goes to results screen.

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/issues/22493
